proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
+
+The main cert-manager repo has 8.4k stars on GitHub and hundreds of projects downstream which depend on cert-manager code in some way. Thousands of people have contributed to the project in some way and cert-manager's use and prominence remains strong.
+
+## Incubation Criteria Checklist
+
+Based on the [graduation criteria][GraduationCriteria], cert-manager seems to clearly tick the boxes required for incubation:
```

Comment:
There's one remaining item on that list: 
> Specifications must have at least one public reference implementation

But that could be the `approver-policy` implementation of 

```
apiVersion: policy.cert-manager.io/v1alpha1
kind: CertificateRequestPolicy
```

Or sample-external-issuer of the external issuer protocol.

Or the sample webhook implementation of the DNS01 webhook protocol

Reply:
I interpreted that "Specifications" section as not being relevant to us, which was why I left it out.

I felt like it was more for projects like [CNI](https://github.com/containernetworking/cni) for whom the spec is a primary goal of the project. In my view, cert-manager doesn't provide a spec; rather, it provides APIs which people can interact with.